scss/components/_soe_dm_article.scss

```diff
@@ -397,8 +401,6 @@
         &[class*="mag-article-color-"] a {
           color: $primary-color;
           text-decoration: underline;
-          font-weight: 600;
-
```

Comment:
Hi @minorwm,

I'm curious why you're deleting this font-weight?
I don't see any reference to it in the description for the PR